etc/inc/disable-programs.inc

```diff
@@ -18,6 +18,7 @@ deny  ${HOME}/.*coin
 deny  ${HOME}/.8pecxstudios
 deny  ${HOME}/.AndroidStudio*
 deny  ${HOME}/.Atom
+deny  ${HOME}/.clion*
```

Comment:
disable-programs.inc is sorted by byte order i.e. this should be round line 68 (depending which lineno you use).